src/pmpo_fortran.f90

```diff
@@ -77,6 +118,27 @@ subroutine polympo_getMPVelArray(mpMesh, n, array) &
     type(c_ptr), value :: array
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief enable the set mesh. The mesh is decleared uneditable all the time.
+  !>        if changes/settings are needed, call this function first, then call
+  !         set functions
```

Comment:
```suggestion
  !> @brief Enable the setting of mesh topology (number of entities and entity adjacencies). 
  !>        By default, modifying the mesh topology without calling this function first will result
  !>        in a runtime failure.
```

We may also want to list which functions require meshEdit = true.

Reply:
Here is the new commit: https://github.com/SCOREC/polyMPO/commit/27059d373dd03d77f998f7224de6795c48afc8de#